ketcher-autotests/tests/Structure-Creating-&-Editing/Actions-With-Structures/R-group-tool/r-group-fragment-tool.spec.ts

```diff
@@ -53,10 +53,6 @@ async function selectRGroup(page: Page, rgroup: string) {
   await page.locator('button', { hasText: rgroup }).click();
 }
 
-async function clickModalButton(page: Page, button: 'Apply' | 'Cancel') {
-  await page.locator(`input[type="button"][value="${button}"]`).click();
-}
-
```

Comment:
In the future, I suggest refactoring this function so that it returns a button with getByTestId. This way it could be more stayable and  the function could be used in other places as well.